test/e2e/catalog_e2e_test.go

```diff
@@ -1028,30 +1028,29 @@ var _ = Describe("Starting CatalogSource e2e tests", func() {
 				},
 			},
 		}
-
 		source, err := crc.OperatorsV1alpha1().CatalogSources(source.GetNamespace()).Create(context.Background(), source, metav1.CreateOptions{})
 		Expect(err).ShouldNot(HaveOccurred())
 		defer func() {
 			err := crc.OperatorsV1alpha1().CatalogSources(source.GetNamespace()).Delete(context.Background(), source.GetName(), metav1.DeleteOptions{})
 			Expect(err).ShouldNot(HaveOccurred())
 		}()
 
-		// Wait for the CatalogSource to be ready
+		By("waiting for the CatalogSource to be ready")
```

Comment:
Nit: Changing these comments to `By()` calls is not strictly necessary. Nice, though.

Reply:
I had some issues (timeouts, IIRC) when running this test locally. `By()` prints steps and it helps to narrow down the piece of the test which had these issues. They act like docs and logs at the same time.

Reply:
Oh, I know. Just saying that these changes aren't strictly related to the primary code change. That being said, no need to change it.